File: thorium/llmGenerated/testModif.py
import pandas as pd
csv = "data.csv"
table = pd.read_csv(csv)
table.loc[1] = ["Hugo",22,"Lyon","INFO"]
table.iloc[1, 3] = "GMA"
table.at[1, 'age'] = 40
table.to_csv("data2.csv",index=False)

File: thorium/llmGenerated/testModif.R
csv <- "data.csv"
table <- read.csv(csv, stringsAsFactors = FALSE)
table[2,] <- c("Hugo",22,"Lyon","INFO")
table[2, 4] <- "GMA"
table[2, "age"] <- 40
write.csv(table, "data2.csv", row.names=FALSE, quote=FALSE)